src/main/java/se/llbit/json/JsonParser.java

```diff
@@ -309,8 +309,8 @@ private void accept(char c) throws IOException, SyntaxError {
   }
 
   private JsonMember parseMember() throws IOException, SyntaxError {
-    if (in.peek() == Literal.QUOTE_MARK) {
-      String name = parseString();
+    if (in.peek() == Literal.QUOTE_MARK || isValidInKey((char) in.peek())) {
```

Comment:
Maybe use opt-in permissive parsing (#5) by a boolean flag that is passed in the constructor?

```suggestion
    if (in.peek() == Literal.QUOTE_MARK || (this.permissive && isValidInKey((char) in.peek()))) {
```